dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -62,6 +62,14 @@ class RayActivityResponse(BaseModel, extra=Extra.allow):
             "This is in the format of seconds since unix epoch."
         ),
     )
+    last_activity_at: Optional[float] = Field(
```

Comment:
why not datetime?

Reply:
All the timestamps from the dashboard endpoints are in seconds to prevent ambiguity if timezones are not correctly provided